content-learn/mastering-meshery-istio/introduction-to-service-meshes/linkerd/dashboard.mdx

```diff
@@ -0,0 +1,101 @@
+---
+docType: "Chapter"
+chapterTitle: "Linkerd Dashboard"
+description: "Meshery is the service mesh managemen plane which offers lifecycle, configuration, and performance management of service meshes and their workloads."
+videos: 4
+lectures: 12
+---
+
+import { ChapterStyle } from "../../../../src/components/Learn-Components/Chapters-Style/chapters.style.js";
+import Top from "../../../../src/assets/images/learn-layer5/linkerd/top.png";
+import Tap from "../../../../src/assets/images/learn-layer5/linkerd/tap.png";
+
+<ChapterStyle>
+
+<h2 class="chapter-sub-heading">Introduction to Linkerd Dashboard</h2>
+
+The Dashboard provides a clickable user interface for administration of Linkerd. The Dashboard provides measurements of success rate, requests/second and latency for services on the mesh. Run the Linkerd Dashboard, by executing:
+
+```sh
+linkerd dashboard &
+```
```

Comment:
@leecalcote is this a typo or do we actually need a `&`?

Reply:
Great question. We actually need `&`.